XIV on Mac/de.lproj/Main.strings

```diff
@@ -633,112 +633,112 @@
-"piU-pX-vSl.title" = "If your game is freezing at startup, \"Compiling Shaders…\" forever, missing lighting or randomly freezing, the problem may be with your shader cache. Deleting it will cause a new one to be rebuilt, though it may result in occasional stutters during play initially.";
+"piU-pX-vSl.title" = "Wenn dein Spiel beim Start einfriert, \"Compiling Shaders...\" ewig dauert, die Beleuchtung fehlt oder das Spiel zufällig einfriert, liegt das Problem möglicherweise an deinem Shader-Cache. Wenn du ihn löschst, wird ein neuer Cache erstellt, auch wenn dies anfangs zu gelegentlichen Stottern beim Spielen führen kann.";
 
 /* Class = "NSMenuItem"; title = "Find Next"; ObjectID = "q09-fT-Sye"; */
-"q09-fT-Sye.title" = "Find Next";
+"q09-fT-Sye.title" = "Nächste finden";
 
 /* Class = "NSButtonCell"; title = "Launch Vanilla Game"; ObjectID = "qDd-CO-Fjs"; */
-"qDd-CO-Fjs.title" = "Launch Vanilla Game";
+"qDd-CO-Fjs.title" = "Unmodifiziertes Spiel starten";
 
 /* Class = "NSTextFieldCell"; title = "Logging In"; ObjectID = "qM7-gN-uyd"; */
-"qM7-gN-uyd.title" = "Logging In";
+"qM7-gN-uyd.title" = "Einloggen";
 
 /* Class = "NSMenuItem"; title = "Windows"; ObjectID = "qRO-Pi-PbC"; */
 "qRO-Pi-PbC.title" = "Windows";
 
 /* Class = "NSButtonCell"; title = "Maximum Frame Rate"; ObjectID = "qVB-Tz-GDg"; */
-"qVB-Tz-GDg.title" = "Maximum Frame Rate";
+"qVB-Tz-GDg.title" = "Maximale Bildrate";
 
 /* Class = "NSTextFieldCell"; title = "This is the special Steam-only Windows License"; ObjectID = "rFx-Wl-XRP"; */
-"rFx-Wl-XRP.title" = "This is the special Steam-only Windows License";
+"rFx-Wl-XRP.title" = "Dies ist die spezielle Steam-only Windows-Lizenz";
 
 /* Class = "NSMenuItem"; title = "Microsoft .NET 7.0"; ObjectID = "rPG-F8-HwW"; */
 "rPG-F8-HwW.title" = "Microsoft .NET 7.0";
 
 /* Class = "NSMenuItem"; title = "Check Spelling While Typing"; ObjectID = "rbD-Rh-wIN"; */
-"rbD-Rh-wIN.title" = "Check Spelling While Typing";
+"rbD-Rh-wIN.title" = "Rechtschreibung während des Tippens prüfen";
 
 /* Class = "NSMenuItem"; title = "Smart Dashes"; ObjectID = "rgM-f4-ycn"; */
-"rgM-f4-ycn.title" = "Smart Dashes";
+"rgM-f4-ycn.title" = "Intelligente Striche";
 
 /* Class = "NSTabViewItem"; label = "Done"; ObjectID = "sdf-Re-eaB"; */
-"sdf-Re-eaB.label" = "Done";
+"sdf-Re-eaB.label" = "Fertig";
 
 /* Class = "NSMenuItem"; title = "Edit Registry"; ObjectID = "snW-S8-Cw5"; */
-"snW-S8-Cw5.title" = "Edit Registry";
+"snW-S8-Cw5.title" = "Registry bearbeiten";
 
 /* Class = "NSMenuItem"; title = "Data Detectors"; ObjectID = "tRr-pd-1PS"; */
-"tRr-pd-1PS.title" = "Data Detectors";
+"tRr-pd-1PS.title" = "Daten-Detektive";
 
 /* Class = "NSButtonCell"; title = "Frame Time Graph"; ObjectID = "ti2-f2-LWe"; */
-"ti2-f2-LWe.title" = "Frame Time Graph";
+"ti2-f2-LWe.title" = "Rahmenzeitdiagramm";
```

Comment:
I would suggest "Frametime Diagramm"; "Rahmenzeitdiagramm" is a literal translation.